src/components/Table/GridTable.stories.tsx

```diff
@@ -93,6 +93,39 @@ export const Hovering = newStory(
   { decorators: [withRouter()] },
 );
 
+export const ActiveCell = newStory(
+  () => {
+    const nameColumn: GridColumn<Row> = {  header: "Name", data: ({ name }) => name };
+    const valueColumn: GridColumn<Row> = { name: "value", header: "Value", data: ({ value }) => value };
+    const actionColumn: GridColumn<Row> = { name: "actions", header: "Action", data: () => <div>Actions</div> };
+    const rowStyles: GridRowStyles<Row> = useMemo(
+      () => ({
+        data: {
+          onClick: (row, api) => {
+            api.setActiveRowId(`data_${row.id}`);
+          },
+        },
```

Comment:
**question:** Is this necessary? Won't the `cellOnClick ` in the GridRow do this for us?

Reply:
on this story I'm doing both active cell and active Row this onClick is just for setting active row

Reply:
:thinking: should we have this line do both:

```
const cellOnClick = applyCellHighlight ? () => api.setActiveCellId(cellId) : undefined;
```

Set the active cell id as well as the active row id?

My initial thought is that this seems great, b/c it starts to move control of "what is the active row?" into GridTable, instead of being something each page needs to know about / reimplement.

Reply:
Y es, we can do that

Reply:
@stephenh what if we only want to highlight a cell, but don't want to use active row styles? I believe this is the case on my ticket

Reply:
Sorry, I misread that code, I thought you were setting the active cell in that `onClick` callback. I'm good with them being treated separately.

Reply:
@JonnCharpentier ah I see, merely setting the active row id turns on active row styles?

I wonder if we need some sort of `rowHighlight`, similar to `cellHighlight`, that tables need to set to explicitly turn on row highlighting behavior...

We can probably deal with that later if you'd like to leave the current `cellOnClick` as-is.